WhaleSpotting/react-app/src/App.tsx

```diff
@@ -13,7 +13,7 @@ import ReportSighting from "./components/ReportSighting";
 function App(): JSX.Element {
     return (
         <Fragment>
-            <Navbar />
+            <Navbar />         
```

Comment:
you seem to have added a load of whitespace here?

Reply:
whitespace removed